pkg/interface/src/views/apps/chat/app.tsx

```diff
@@ -194,6 +194,11 @@ export default class ChatApp extends React.Component<ChatAppProps, {}> {
             render={(props) => {
               let station = `/${props.match.params.ship}/${props.match.params.station}`;
 
+              // ensure we know joined chats
+              if(!chatInitialized) {
+                return null;
+              }
```

Comment:
Wouldn't this just blank screen people leaping to an unmanaged chat during first session? If we just no-op the join on back-end do we need this?

Reply:
It'll blank screen them until it loads, ensuring the leap still works

Reply:
is chatInitialised tied to a reactive prop?

Reply:
yeah